lms/services/lti_h.py

```diff
@@ -3,7 +3,7 @@
 from lms.services import HAPIError
 
 
-class LTIHService:
+class LTIHService:  # pylint:disable=too-few-public-methods
```

Comment:
I think we may want to globally disable this pylint warning

Reply:
Yeah I agree. It's clearly part of our style